### DISCUSSION TITLE
Contabilizzazione giornaliera dei consumi

### DISCUSSION THREAD

**Discussion by baurbr** [indomus/forum#147] [HUB: Home Assistant]
Ciao a tutti...
ho due shelly1 PM collegati a dei scaldasalviette in bagno. Vorrei avere una statistica di consumo giornaliera su HA. Mi date qualche suggerimento su come realizzarla e avere sulla plancia un grafico che visualizzi l'andamento dei consumi giorno per giorno?  Grazie di cuore

**Comment by indomus** [comment]
https://indomus.it/progetti/configurare-dei-contabilizzatori-di-consumo-su-home-assistant/

**Reply by baurbr** [reply]
Grazie, avevo già visto la guida... Non so però come inserire in HA un grafico che mi faccia vedere il consumo giornaliero nel tempo. Si può fare ?

**Reply by xyzmauro** [reply]
> Grazie, avevo già visto la guida... Non so però come inserire in HA un grafico che mi faccia vedere il consumo giornaliero nel tempo. Si può fare ?

Ciao, guarda questa custom card 
https://github.com/kalkih/mini-graph-card

**Reply by baurbr** [reply]
Grazie!!!

**Reply by xyzmauro** [reply]
Se guardi nell'altra discussione dove hai chiesto come si fa a fare quel tipo di grafico c'è anche l'esempio 😉

**Reply by baurbr** [reply]
Come si dice dalle mie parti... "che mona che sono"
